lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.scss

```diff
@@ -10,13 +10,13 @@
     &-footer.mat-card-footer {
         margin: 0;
         padding: 8px 12px;
-        border-top: 1px solid var(--adf-theme-foreground-text-color-007);
+        border-top: 1px solid var(--adf-theme-foreground-text-transparent-opacity);
 
         button {
-            color: var(--adf-theme-foreground-text-color-054);
+            color: var(--adf-theme-foreground-text-dark-opacity);
```

Comment:
We will loose effect of hover. Is it intentional? button color may be 
--adf-theme-foreground-text-light-opacity

Reply:
Checking seemed fine, but i can use the variable you suggested. I was trying to group same variables with similar values together